### ISSUE TITLE
'Run This Tutorial' does not work.

### ISSUE THREAD

**Issue by Welding-Torch** [stateful/blog-examples#12]
_Hi devs,_

So I just came across RUNME from Hacker news today, and I decided to install the VS Code extension for it and follow along with the 'Get Started' tutorial. But I encountered an issue while attempting to run the first 'Run This | Readme'
 badge example provided by Getting Started guide linked in the 'runme' extension page. I instantly came across an undescribed error and a breaking issue. I will describe them here.

### 1) Undescribed Error (Can Be Fixed By Typing 'yes')

![image](https://github.com/stateful/blog-examples/assets/46340124/d2a46606-fe5f-4c5c-9485-d646194ef334)

This was the first error that I came across, and it's very simple, I just had to type yes and it proceeded. However, this is something that the devs should be aware of, and possibly add to the [README.MD] in order to make sure that the reader knows what to expect. (https://github.com/stateful/blog-examples/blob/main/node-streams/README.md)

### 2) Breaking Issue ( **Needs Urgent Solution** )
![image](https://github.com/stateful/blog-examples/assets/46340124/18ac2966-7fb0-49e2-93c0-0c1b05f6cc2f)

Output:
```
Executing task: git clone git@github.com:stateful/blog-examples.git "c:\Users\bhati\AppData\Roaming\Code\User\globalStorage\stateful.runme\stateful\blog-examples"

Cloning into 'c:\Users\bhati\AppData\Roaming\Code\User\globalStorage\stateful.runme\stateful\blog-examples'...
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights and the repository exists.

The terminal process "C:\WINDOWS\System32\cmd.exe /d /c git clone git@github.com:stateful/blog-examples.git "c:\Users\bhati\AppData\Roaming\Code\User\globalStorage\stateful.runme\stateful\blog-examples"" terminated with exit code: 128.
Terminal will be reused by tasks, press any key to close it.

``` 

This stopped me from continuing with the tutorial. As you can see, it refuses to let me clone the repository. Perhaps it is an SSH issue? Hoping for a fix 🤞 

---
### Steps to Reproduce:
To reproduce this error, merely trying install runme on a fresh installation of Windows and VS Code, and try to run the badge in 'Getting Started' (blog-examples/main/node-streams/README.md).

Thank you for reading, have a great day.
❤️

**Comment by admc** [comment]
@Welding-Torch thanks for letting us know, we'll look into it ASAP!

**Comment by christian-bromann** [comment]
Thanks for reporting @Welding-Torch , I will take a took as soon as I can.

**Comment by christian-bromann** [comment]
Thanks for reporting @Welding-Torch , I will take a look at soon as I can.

**Comment by christian-bromann** [comment]
@Welding-Torch thanks again for your input. I looked into both queries:

> 1) Undescribed Error (Can Be Fixed By Typing 'yes')

This is not an error, it is more a security feature that ensures you are ok connecting to a remote server in order to get the code. However with the fix I applied this message should go away.

> 2) Breaking Issue ( **Needs Urgent Solution** )

The git url we used (`git@github.com:stateful/blog-examples.git`) required git to authenticate with Github which in your case failed. I've updated the docs to use a HTTPS link which doesn't require authentication.

Can I ask you to try again and see if it all works? Thank you!
